src/api/index.ts

```diff
@@ -0,0 +1,38 @@
+import express from 'express'
+import morgan from 'morgan'
+import helmet from 'helmet'
+import cors from 'cors'
+import { Environment } from '../scripts'
+import { Repository } from 'typeorm'
+import { ScheduledExecution } from '../entities'
+import getTransactionHashByExecutionId from './getTransactionHashByExecutionId'
+
+const app = express()
+app.use(morgan('dev'))
+app.use(helmet())
+app.use(cors())
+
+const startApi = (environment: Environment, repository: Repository<ScheduledExecution>) => {
```

Comment:
```suggestion
const setupApi = (app: Express.App, repository: Repository<ScheduledExecution>) => {
```

We need to instantiate `app` in the src/index.ts and pass it as a parameter to be able to add unit tests to the API. We can use a tool like `supertest`. Also set middlewares and listen in the src/index.ts